lib/gc/Transforms/Pipeline.cpp

```diff
@@ -143,6 +144,8 @@ class GCCPUPipeline : public impl::GCCPUPipelineBase<GCCPUPipeline> {
     auto op = getOperation();
     PassManager pm{op->getContext()};
     populateCPUPipeline(pm);
+    // TODO(longsheng): add a option to
+    // disable threading and enable pm.enableIRPrinting();
```

Comment:
What was this TODO comment for previously ?

Reply:
There were no code here previously, I use to add pm.enableIRPrinting() for debug internal passes,  but this function also requires disabling threading, we can figure out later.